docs/guides/mirror_management/add_mirror_manager.md

```diff
@@ -15,7 +15,7 @@ Please note that we are not allowed to accept public mirrors in countries subjec
 
 As of this writing (late 2022), storage space requirements for mirroring all current and past Rocky Linux releases is about 2 TB.
 
-Our master mirror is `rsync://msync.rockylinux.org/rocky/mirror/pub/rocky/`
+Our master mirror is `rsync://msync.rockylinux.org/rocky/mirror/pub/rocky/` .
```

Comment:
without a period here, the next word should be lowercase.
What do you think?

Reply:
@SergeCroise This is correct, **BUT** again, you've added a space after the "`" and before the "."